server/lib/fetch.js

```diff
@@ -0,0 +1,42 @@
+import fetch from 'node-fetch';
+
+import logger from './logger';
+
+/**
+ * Make a fetch call with a timeout. Returns a thenable Promise.
+ * @param {Object} fetchParams The object of options to pass into the fetch request.
+ * @param {String} fetchParams.url The url to fetch.
+ * @param {Object} [fetchParams.options] The method, headers, and other options for node-fetch. Default method is GET.
+ * @param {Number} [timeoutInMs] The callback to call with the authorized client. Default is 5000 ms.
+ */
+
+export const fetchWithTimeout = (fetchParams, timeoutInMs = 5000) => {
```

Comment:
I think it would look better by following `fetch` API: first argument `url`, second argument `options`. `timeoutInMs` could be read from `options` or kept as third argument.